core/services/form/meta/InputTypes.php

```diff
@@ -2,6 +2,7 @@
 
 namespace EventEspresso\core\services\form\meta;
 
+use EventEspresso\core\services\form\meta\inputs\Block;
```

Comment:
i might move this out of the `inputs` folder/namespace since it's not technically an input type...
will leave as is for now, until i think of how i want to approach this.
thnx for adding it though  as it's obviously something i missed 👍🏻